src/Symfony/Component/Config/Resource/ResourceInterface.php

```diff
@@ -31,6 +37,9 @@ public function __toString();
      * @param int $timestamp The last time the resource was loaded
      *
      * @return bool True if the resource has not been updated, false otherwise
+     *
+     * @deprecated since 2.8, to be removed in 3.0. If your resource can check itself for
+     *             freshness implement the SelfCheckingResourceInterface instead.
```

Comment:
keeping the method in this interface means that people are still forced to implement it though, which is bad as it cannot be done.